openai.el

```diff
@@ -82,6 +82,9 @@ auth-source is provided for convenience.")
   "A unique identifier representing your end-user, which can help OpenAI to
 monitor and detect abuse.")
 
+(defvar openai-base-url "https://api.openai.com/v1"
+  "The base URL for OpenAI API requests.")
```

Comment:
I think this can be `defcustom`?

Reply:
The `openai-completion-max-tokens ` and `openai-completion-temperature ` defcustoms are defined inside the module files. Should I do the same for the `openai-base-url`.

Reply:
`defcustom` means we expect user to customize this variable. `defvar` is just a global variable internally used inside this module. `openai-key` was using `defcustom` but it has changed due to the following reason:

> 3. `openai-key` should not be defcustom. Many users commit their custom file and they could accidentally commit their tokens.

Source: https://github.com/emacs-openai/openai/issues/5#issue-1611978573

I guess I've accidentally changed `openai-user` to `defvar`... But let's ignore this for now. I will change this back when I have time. :D

For `openai-base-url`, it depends if we expect user to customize this or not. In terms of functionality, it doesn't create any other side effects.